test_runner/regress/test_lfc_prewarm.py

```diff
@@ -70,10 +136,20 @@ def test_lfc_prewarm(neon_simple_env: NeonEnv):
     assert cur.fetchall()[0][0] == n_records * (n_records + 1) / 2
 
     check_pinned_entries(cur)
+    if with_compute_ctl:
+        desired = {
+            "status": "completed",
+            "total": total,
+            "prewarmed": prewarmed,
+            "skipped": skipped,
+        }
+        assert http_client.prewarm_lfc_status() == desired
+        assert prom_parse(http_client) == {OFFLOAD_LABEL: 0, PREWARM_LABEL: 1}
 
 
 @pytest.mark.skipif(not USE_LFC, reason="LFC is disabled, skipping")
-def test_lfc_prewarm_under_workload(neon_simple_env: NeonEnv):
+@pytest.mark.parametrize("with_compute_ctl", [False, True], ids=["", "compute-ctl"])
+def test_lfc_prewarm_under_workload(neon_simple_env: NeonEnv, with_compute_ctl: bool):
```

Comment:
Can you please add test comments clarifying what parameters mean and what are the different test modes? After quickly eyeballing the test, I cannot easily grasp the `with_compute_ctl` and why we pass `ids` as a test parameter

Reply:
Also, please don't use an empty string for an ID.